server/routes/auth.js

```diff
@@ -0,0 +1,55 @@
+const router = require("express").Router();
```

Comment:
I'm noting many of the styling inconsistencies indicated above (quote type, spacing, indentation issues, inconsistent use of space before object values, etc.) within all of the routes files as well.